packages/blade/src/components/Badge/__tests__/Badge.web.test.tsx

```diff
@@ -16,7 +16,7 @@ const colors: BadgeProps['color'][] = [
 ];
 
 describe('<Badge />', () => {
-  it('should render Badge with default props', () => {
+  it('should render Badge with primary props', () => {
```

Comment:
global replace issue?
```suggestion
  it('should render Badge with default props', () => {
```

Reply:
fixed